linode/manager
# chore: Repo Clean Up - Remove Old Redux Docs

## Description 📝

- We had some old system that used redux to display some extra documentation, we haven't used this in the last few years so the PR removes it and does some other random clean up

## Preview 📷

- There should be no visual differences

## How to test 🧪

- Verify all automated tests pass
- Visit some of the components that may have changed by viewing the diff and verify that things work normally
